primer3/src/libprimer3/read_boulder.c

```diff
@@ -206,13 +207,14 @@ read_boulder_record(FILE *file_input,
                     pr_append_str *nonfatal_parse_err,
                     pr_append_str *warnings,
                     read_boulder_record_results *res,
-                    char* in_buffer
+                    char* in_buffer /* primer3-py new argument */
 ) {
   int line_len;
   int tag_len, datum_len;
   int data_found = 0;
   int read_start_codon = 0;
-  char *s, *n, *datum, *task_tmp = NULL;
+  char *n, *datum, *task_tmp = NULL;
+  char* s = NULL;
```

Comment:
Nit: `char* s` -> `char *s` for consistency

Reply:
Updating to `char *` instead of `char*` for consitency but I prefer the latter for readability